modules/cluster-resources.adoc

```diff
@@ -0,0 +1,39 @@
+[id="support-cluster-resources_{context}"]
+= Interacting with your cluster resources
+
+You can interact with cluster resources using OpenShift CLI (`oc`) in {product-title}. The cluster resources that you see after running `oc api-resources` command are editable.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* Access to the web console or the `oc` installed.
```

Comment:
These are sentence fragments and thus wouldn't need periods. But generally, I prefer making them complete sentences (your call):
```suggestion
* You have access to the cluster as a user with the `cluster-admin` role.
* You have access to the web console or you have installed the `oc` CLI tool.
```

Reply:
acked